backup/CMakeLists.txt

```diff
@@ -2,16 +2,11 @@ cmake_minimum_required(VERSION 2.8.8)
 project(HotBackup)
 
 # pick language dialect
-check_cxx_compiler_flag(-std=c++03 HAVE_STDCXX03)
-if (HAVE_STDCXX03)
-  set(CMAKE_CXX_FLAGS "-std=c++03 -Wno-deprecated-declarations ${CMAKE_CXX_FLAGS}")
+check_cxx_compiler_flag(-std=c++11 HAVE_STDCXX03)
```

Comment:
Check for -std=c++11 and but set HAVE_STDCXX03.
All this snippet must go away instead, because the MySQL CMake itself now checks for C++11.

Reply:
Oops, didn't change the macro indicator.   This library is used but 5.6, 5.7, and was part of Mongo products, so shouldn't totally assume 8.0 yet.